lib/src/track/options.dart

```diff
@@ -1,69 +1,195 @@
+import '../imports.dart';
+
 /// Options when creating a LocalVideoTrack.
 class LocalVideoTrackOptions {
-  CameraPosition position = CameraPosition.front;
-  VideoParameter params;
-
-  LocalVideoTrackOptions({
-    VideoParameter? params,
-    CameraPosition? position,
-  }) : params = VideoPresets.qhd {
-    if (params != null) {
-      this.params = params;
-    }
-    if (position != null) {
-      this.position = position;
-    }
-  }
-
-  Map<String, dynamic> get mediaConstraints {
-    return <String, dynamic>{
-      'mandatory': params.mediaConstraints,
-      'facingMode': position == CameraPosition.front ? 'user' : 'environment',
-    };
-  }
+  //
+  final CameraPosition position;
+  final VideoPreset params;
```

Comment:
we shouldn't change this to VideoPreset. A preset is a set of params that we've provided. If users are creating their own, it will no longer be a preset.

Reply:
Makes sense!

Reply:
Fixed in b78ded11e976423787def34cf5c4cfabb6928509